proposals/AlphaProposal.sol

```diff
@@ -0,0 +1,100 @@
+pragma solidity ^0.8.0;
+
+import {Strings} from "@openzeppelin/utils/Strings.sol";
+
+import {VmSafe} from "@forge-std/Vm.sol";
+
+import {Proposal} from "@proposals/Proposal.sol";
+import {Addresses} from "@addresses/Addresses.sol";
+
+abstract contract AlphaProposal is Proposal {
```

Comment:
wouldn't it be better to implement this directly in the Proposal class for a cleaner solution?